salt/config/__init__.py

```diff
@@ -1579,7 +1579,7 @@ def _gather_buffer_space():
     'sign_pub_messages': True,
     'add_proxymodule_to_opts': False,
     'proxy_merge_grains_in_module': True,
-    'append_minionid_config_dirs': ['cachedir', 'pidfile'],
+    'append_minionid_config_dirs': ['cachedir', 'pidfile', 'default_include', 'include'],
```

Comment:
Perhaps for this we should only append the minion ID only for `default_include`: https://github.com/saltstack/salt/pull/40291/commits/423dda9149e62720e46a273bab4b3446a27d53bf?